cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -588,6 +588,34 @@
     </description>
   </property>
 
+  <property>
+    <name>app.run.records.table.ttl.age.days</name>
+    <value>0</value>
+    <description>
+      Amount of time (in days) to retain run records for old job runs. Set to 0
+      to retain them indefinitely.
+    </description>
+  </property>
+
+  <property>
+    <name>app.run.records.table.ttl.frequency.hours</name>
+    <value>12</value>
+    <description>
+      How frequently (in hours) to check for old run records. Has no effect if
```

Comment:
'has no effect' is ambiguous, what does it mean? From https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ScheduledExecutorService.html#scheduleAtFixedRate(java.lang.Runnable,%20long,%20long,%20java.util.concurrent.TimeUnit) I think it means it would constantly keep trying to delete?

Reply:
Updated the description here, let me know if this is clearer.

Updated behavior to wait a minimum of 1 hour.